src/base-element.js

```diff
@@ -82,9 +84,20 @@ import {vsyncFor} from './vsync';
  * before buildCallback and it might be called multiple times including
  * after layoutCallback.
  *
- * The documentInactiveCallback is called when the document becomes
- * inactive. E.g. when the user swipes away from the document or
- * focuses a different tab.
+ * The pauseCallback is called when when the document becomes inactive, e.g.
+ * when the user swipes away from the document, or when the element is no
+ * longer being displayed, e.g. when the carousel slide slides out of view.
+ * In these situations, any actively playing media should pause.
+ *
+ * The resumeCallback is called when when the document becomes active again
+ * after becoming inactive, e.g. when the user swipes away from the document
+ * and swipes back. In these situations, any paused media may begin playing
+ * again, if user interaction is not required.
+ * // TODO(jridgewell) slide slides into view
+ *
+ * The unlayoutCallback is called when the document becomes inactive, e.g.
```

Comment:
> "is called when the document becomes inactive" reads the same as for the pauseCallback.

That's because they're both called in that situation.

> do we have any example of an element that supports both: pause and unlayout?

There are no elements that use both pause and unlayout.